src/System.Management.Automation/engine/remoting/commands/EnterPSHostProcessCommand.cs

```diff
@@ -345,6 +396,22 @@ private void VerifyProcess(Process process)
             }
         }
 
+        private void VerifyPipeName(string customPipeName)
+        {
+            // Named Pipes are represented differently on Windows vs macOS & Linux
+            var pipePath = Platform.IsWindows ? $@"\\.\pipe\{customPipeName}" : Path.Combine(Path.GetTempPath(), $"CoreFxPipe_{customPipeName}");
```

Comment:
I don't like interpolate operator because it implicitly uses CurrentCulture but we always prefer set explicit value. Could we please use `string.Format`?

Reply:
moved to StringBuilder.